src/app/(Web)/wedding/invitations/_components/InvitationFloating/InvitationFloating.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+
+import { InvitationTemplate } from '@/components/server';
+import { QUERY_OPTIONS } from '@/constants';
+import useClickAway from '@/hooks/useClickAway/useClickAway';
+import { useQuery } from '@tanstack/react-query';
+
+import { InvitationFloatingProps } from './InvitationFloating.type';
+
+const InvitationFloating = ({ id }: InvitationFloatingProps) => {
+  const route = useRouter();
+  const ref = useClickAway<HTMLDivElement>(() => route.back());
+  const { data } = useQuery(QUERY_OPTIONS.INVITATION(id));
+
+  if (!data?.result) {
+    return;
+  }
+
+  return (
+    <div
+      className='flex flex-col justify-center items-center gap-[2rem]'
+      ref={ref}
+    >
+      <InvitationTemplate
+        title={data.result.title}
+        details={data.result.details}
+        groomName={data.result.groomName}
+        brideName={data.result.brideName}
+        productInfo={data.result.productInfo}
+        width={200}
+        height={100}
+      />
+      <Link
+        href={`wedding/invitations/produce${id}`}
```

Comment:
"/" 혹은 쿼리 스트링 표기 누락인가효? 아니면 이렇게 하더라도 정상 동작하나요?

```suggestion
        href={`wedding/invitations/produce/${id}`}
```